src/tests/test_pyodide.py

```diff
@@ -3,7 +3,7 @@
 import sys
 from textwrap import dedent
 
-sys.path.append(str(Path(__file__).parents[2] / "src" / "pyodide-py"))
+sys.path.append(str(Path(__file__).absolute().parents[2] / "src" / "py"))
```

Comment:
Path.absolute is not a documented/public method in https://docs.python.org/3/library/pathlib.html 

```suggestion
sys.path.append(str(Path(__file__).resolve().parents[2] / "src" / "py"))
```
same for `src/tests/test_console.py`

Reply:
Huh I never noticed.
```
>>> print(pathlib.Path.absolute.__doc__)
Return an absolute version of this path.  This function works even if the path doesn't point to anything.

No normalization is done, i.e. all '.' and '..' will be kept along. Use resolve() to get the canonical path to a file.
```

Reply:
https://discuss.python.org/t/pathlib-absolute-vs-resolve/2573